contributor_path.rst

```diff
@@ -0,0 +1,212 @@
+
+The goal of the path, first of all, is to make your activity more determined and efficient through understanding of what you can achieve and how. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. When you contribute regularly, your proficiency and judgement in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+Regular contribution makes you a member of the community and leads to respect of other community members.
+
+Be persistent and act gradually. Great result is just a product of time and effort. Here's how to get started.
+
+.. note::
+
+   If you have any questions, feel free to reach out to the community in the ``#ansible-community`` `Libera.Chat <https://libera.chat/>`_ IRC channel. If you have ideas on how this document can be improved, please, share them through creating an issue or a pull request.
+
+.. contents::
+
+What is a contribution to an Open Source project
+================================================
+
+First, we should understand what a contribution to an Open Source project is.
+
+The contribution to an Open Source project is any positive addition or change given by an individual or a company to the project including increasing its functionality, improvements to the codebase, code developer experience, user experience, increasing the project's popularity and community, providing infrastructure, and any other kinds of active social or material support.
+
+Note that, according to the classification given in the `Contribution to an Open Source project guide <contribution_to_project.rst>`_, most types of contribution do not require programming skills.
+
+Determine areas of your interests
+=================================
+
+First, determine areas that are interesting to you taking into consideration your current experience and experience you'd like to gain.
+There are some ideas about how you can contribute to the project listed in the `Contribution guide <contribution_to_project.rst>`.
+
+Determine the project
+=====================
+
+The Ansible ecosystem consists of the following projects:
+
+  - `Ansible Core <https://docs.ansible.com/ansible-core/devel/index.html>`_
+  - `Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+  - `AWX <https://github.com/ansible/awx>`_
+  - `Galaxy <https://galaxy.ansible.com/>`_
+  - `Lint <https://ansible-lint.readthedocs.io/en/latest/>`_
+  - `Molecule <https://molecule.readthedocs.io/en/latest/>`_
+
+You can contribute to any of them.
+
+Learn
+=====
+
+The required skillset depends on the area of interest and the project chosen previously.
+
+Some of the `ways <contribution_to_project.rst>`_ to contribute do not require technical skills and some of them imply that you will invest time in learning.
```

Comment:
This is a repeat from line 36. Is it necessary ?

Reply:
I wouldn't say this is literal repetition:) The goals of this sentence are:
- highlight once more that even if you're not a programmer, you can find an application for yourself in the project
But alright, let's get rid of this partial duplication:)